__tests__/integration.test.js

```diff
@@ -368,3 +368,38 @@ describe('404: error handling invalid endpoints', () => {
       });
   });
 });
+
+describe('/api/articles/:article_id adding comment_count', () => {
+  test('respond with a 200 status code with an individual article object with CommentCount ', () => {
+    return request(app)
+      .get('/api/articles/2')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          article_img_url: expect.any(String),
+          comment_count: expect.any(String),
```

Comment:
In the above tests for this same endpoint you are already testing for all of the other properties here, the only new property is comment_count so thats all we really need to test for here.